src/Microsoft.DotNet.XHarness.InstrumentationBase.Xunit/DefaultAndroidEntryPoint.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using Microsoft.DotNet.XHarness.TestRunners.Common;
+using Microsoft.DotNet.XHarness.TestRunners.Xunit;
+
+namespace Microsoft.DotNet.XHarness.DefaultAndroidEntryPoint.Xunit
```

Comment:
Can you please add a documentation header explaining the purpose of this class?
I think it will be very important for anyone looking into this.

Also please make a note in the README, where we talk about TestRunners. Otherwise people who are not Matthew won't know about this

Reply:
added